pom.xml

```diff
@@ -58,20 +67,6 @@
 			</dependency>
 		</dependencies>
 	</dependencyManagement>
-	<properties>
-		<revision>1.0.0</revision>
-		<changelist>-SNAPSHOT</changelist>
-		<jenkins.version>2.319.1</jenkins.version>
-		<hpi-plugin.version>3.25</hpi-plugin.version>
-	</properties>
-	<licenses>
-		<license>
-			<name>Apache License, Version 2.0</name>
```

Comment:
What's the reason to remove the license section?

Reply:
I wouldn't. It was moved to https://github.com/jenkinsci/crowdstrike-security-plugin/pull/1/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R23-R30 using `mvn tidy:pom` to re-order `pom.xml`